src/main/java/core/basesyntax/TreeSetCharacters.java

```diff
@@ -11,13 +17,34 @@
  * про відсутність файлу.</p>
  *
  * <p>Приклад 1: ur-BvT?^ ra w; p
- * Результат 1: abprt</p>
+ * Результат 1: abprt                   </p>
  *
  * <p>Приклад 2: A _f*c a?F
- * Результат 2: acf</p>
+ * Результат 2: acf                      </p>
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+    public static final int MAX_LENGTH_LETTER = 5;
+
+    public String getUniqueCharacters(String fileName) throws IOException {
+        Set<Character> treeSet = new TreeSet<>();
+        try (FileReader fileReader = new FileReader(fileName)) {
+            int character = fileReader.read();
```

Comment:
```suggestion
            int index= fileReader.read();
```

Reply:
@andriylagun I would say, `character` suits better because what we actually read is character codepoint